### PR TITLE
RYA-529 fixed boundless and locationtech repos in main pom in Rya v4

### PR DESCRIPTION
**Pull request by DLotts**
Replaced boundless and locationtech repos in main pom in Rya v4
another pull request does the same for the v3 branch.

Tests: Full build succeeded

See issue here: [Jira](https://issues.apache.org/jira/browse/RYA-529)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -247,7 +247,7 @@ under the License.
                 </repository>
                 <repository>
                     <id>LocationTech - Third Party</id>
-                    <url>https://repo.locationtech.org/content/repositories/thirdparty/</url>
+                    <url>https://repo.eclipse.org/content/repositories/thirdparty/</url>
```
Comment (adinancr) [inline pom.xml:250]: This should be:  https://repo.eclipse.org/content/repositories/locationtech-thirdparty/

Comment (DLotts) [reply pom.xml:250]: Good catch!

Comment (adinancr) [reply pom.xml:250]: LGTM. Do you want to merge this?

Comment (DLotts) [reply pom.xml:250]: sure.  I'll try to do it today.

**Pull request merged**